project.clj

```diff
@@ -7,10 +7,13 @@
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.9.0"]
                  [org.clojure/core.async "0.4.474"]
+                 [clj-time "0.14.2"]
                  [cheshire "5.8.0"]
+                 [jarohen/chime "0.2.2"]
```

Comment:
We don't need to add chime or clj-time here - we can use core.async `(timeout)` and java.Date